knexfile.js

```diff
@@ -0,0 +1,54 @@
+// Update with your config settings.
+const localPg = {
+  host: 'localhost',
+  database: 'productionDB',
+  user: 'Omar',
+  password: 'pass',
+};
+
+const productionDbConnection = process.env.DATABASE_URL || localPg;
```

Comment:
this portion (and the localPg) was only relevant if wanting to migrate to postgresql for heroku, but good to be in the habit of adding it in now and possibly deploying later than forgetting entirely before deployment